### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by VedanthB**
- added modal classes 

https://user-images.githubusercontent.com/75097551/152438896-4723c0c5-16d5-4f5c-affb-c3c644c1b976.mov

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modal-content {
+  position: relative;
+  margin: auto;
+  border: 1px solid #888;
```
Comment (krishanu2911) [inline components/modal.css:20]: A suggestion to use color CSS variable

File: components/modal.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  to {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+/* The Close Button */
+.modal-close {
+  /* float: right; */
+  font-size: 28px;
```
Comment (krishanu2911) [inline components/modal.css:54]: prefer rem over px

**Review by krishanu2911 — COMMENTED** [review]
Added a few suggestions in modal.css
Everything else seems good to go in modal

**Review by ashishnnnnn — APPROVED** [review]
Code looks good to me !!

**Pull request merged**